Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -362,7 +362,27 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
         // Compute the map of known versions.
         //
         // FIXME: Move this utility to a more stable location.
-        let knownVersions = Git.convertTagsToVersionMap(repository.tags)
+        let knownVersionsWithDuplicates = Git.convertTagsToVersionMap(repository.tags)
+        
+        let knownVersions = knownVersionsWithDuplicates.mapValues { (tags) -> String in
+            switch tags {
+            case (normal: .some, vPrefixed: .none):
+                return tags.normal!
```

Comment:
Its generally not a good idea to use ! in swift unless there is a really good reason to do that.

Reply:
I felt weird about doing that for sure, but I thought it would be ok to force unwrap here since I explicitly check the tuple optionals to see if they are .some or .none. This weird switch-case will be gone with the use of arrays though!